sentences/pl/_common.yaml

```diff
@@ -123,10 +361,25 @@ expansion_rules:
   close_blind: "(zasłoń|opuść)"
   close_curtain: "(zasunięte|zasłonięte)"
   brightness: "{brightness}[%| procent]"
+  light: "(światło|lamp(a|ę)|oświetleni(a|e)|żarów(ka|kę))"
+  lights: "(świat(ła|eł)|lamp[y]|oświetleni(a|e)|żarów(ki|ek))"
   temp: "temperatur(a|ę|y)|stopni|zimno|gorąco"
   temperature: "{temperature}[°| stopni] [{temperature_unit}]"
+  numeric_value_set: "(ustaw|zmień|przestaw|nastaw|zwiększ|zmniejsz)"
+  position: "{position}[[ ]%| procent]"
+  volume: "{volume:volume_level}[[ ]%| procent]"
+  # Questions
+  what_is_the_class_of_name: "(<what_is> <class> (w|z|(wskazany|wymierzony)) <name> [<area>]|<what_is> <name> <class> [<area>]|<what_is> <area> <name> <class>)"
```

Comment:
This variable is just boilerplate for all question related with binary_sensor and sensor. Currently it is not use in any question. It will probably be expanded with additional words. I will update it after I will start working with this two type of entities.